### ISSUE TITLE
Proxy Configuration For Soap Client

### ISSUE THREAD

**Issue by umutphp** [vipsoft/JiraExtension#9]
Hi,

I am using the JiraExtension in an intranet project. The Jira is only reachable via a HTTP proxy.  There is no option to configure proxy for Soap client.

**Comment by robocoder** [comment]
The SoapClient is configured via dependency injection.  You can simply modify the service configuration, e.g.,

```
<service id="behat.jira.soap_client" class="SoapClient" public="false">
    <argument type="string">%behat.jira.host%/rpc/soap/jirasoapservice-v2?wsdl</argument>
    <argument type="collection">
        <argument key="proxy_host">proxy</argument>
        <argument key="proxy_port">8080</argument>
        <argument key="proxy_login">my-login</argument>
        <argument key="proxy_password">myPassword</argument>
    </argument>
</service>
```

**Comment by umutphp** [comment]
Is there a way to configure proxy in "behat.yml" file?

**Comment by robocoder** [comment]
Not yet.

**Comment by chrfritsch** [comment]
Would be very awesome to configure proxy with behat.yml

**Comment by andrewlcg** [comment]
Issued pull request https://github.com/vipsoft/JiraExtension/pull/20 to handle this

**Comment by robocoder** [comment]
Fixed in a64548d1f6f64de2f0d3b6a55ed9493c09bf38e1 and 2b746da5d92d8379c2b3b2453d6efd3e2d3d13a3 of 2.5 branch.
